controls/ccn_rhel9.yml

```
@@ -0,0 +1,836 @@
+    original_title: Se auditan los intentos de acceso a elementos críticos.
+    levels:
+      - advanced
+    status: planned
+    related_rules:
+      - audit_rules_unsuccessful_file_modification_creat
+      - audit_rules_unsuccessful_file_modification_ftruncate
+      - audit_rules_unsuccessful_file_modification_open
+      - audit_rules_unsuccessful_file_modification_openat
+      - audit_rules_unsuccessful_file_modification_truncate
+
+  - id: A.3.SEC-RHEL10
+    title: All mount operations on the system and changes to the swap are audited
+    original_title: Se audita toda operación de montaje en el sistema y modificaciones en la memoria de intercambio.
+    levels:
+      - intermediate
+      - advanced
+    status: pending
+    notes: |-
+      We probably have audit related rule to monitor mount related syscalls, but it is not clear
+      about the swap. Is the intention to monitor when swap is changed?
+    related_rules:
+      - audit_rules_media_export
+
+  - id: A.3.SEC-RHEL11
+    title: Modifications in PAM files are audited
+    original_title: Se auditan modificaciones en ficheros PAM.
+    levels:
+      - advanced
+    status: pending
+    notes: |-
+      The intention here is probably to audit changes in /etc/pam.d files, but we need to confirm
+      this assumption and get more context.
+
+  #[A.4] Manipulación de los ficheros de configuración
+  - id: A.4.SEC-RHEL1
+    title: Common users do not have local administrator permissions or are included in a sudo group
+    original_title: Los usuarios estándar no disponen de permisos de administrador local ni se encuentran incluidos en un grupo sudoer.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: pending
+    notes: |-
+      It is a little tricky to interpret this requirement. Assuming the "Common users" are actually
+      interactive users, this requirement would automatically enforce all admin actions to be
+      performed only by the root user. I am not sure if this is the intetion here.
+
+  - id: A.4.SEC-RHEL2
+    title: The system has an updated antivirus
+    original_title: El sistema tiene un antivirus y este está actualizado.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: pending
+    notes: |-
+      New templated rule is necessary to install the package. But to ensure the chosen antivirus
+      is actually updated would demand a more complex rule. Maybe this requirement can have at
+      leastthe partial status after the templated rule.
+
+  - id: A.4.SEC-RHEL3
+    title: Permissions by partitions are modified
+    original_title: Se modifican los permisos por particiones.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: pending
+    notes: |-
+      Related to nosuid, noexec and nodev options but in /boot. More context is needed.
+
+  #[A.5] Suplantación de la identidad.
+  - id: A.5.SEC-RHEL1
+    title: Control login and impersonation permissions
```

```suggestion
    title: Login and impersonation permissions are controlled
```
Modified it for an increased consistency with the rest of the titles.